pkg/web/api.go

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: 2023 Greenbone AG <https://greenbone.net>
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2024 Greenbone AG <https://greenbone.net>
```

Reply:
This suggested change is similar to others

Reply:
Changed everywhere `2023 Greenbone AG` to ` 2024 Greenbone AG`